Mage.Client/src/main/java/mage/client/dialog/NewTournamentDialog.java

```diff
@@ -63,11 +60,15 @@ public class NewTournamentDialog extends MageDialog {
 
     public NewTournamentDialog() {
         initComponents();
+        this.customOptions = new CustomOptionsDialog(CustomOptionsDialog.SaveLoadKeys.TOURNEY, btnCustomOptions);
+        MageFrame.getDesktop().add(customOptions, JLayeredPane.MODAL_LAYER);
         lastSessionId = "";
         txtName.setText("Tournament");
         this.spnNumWins.setModel(new SpinnerNumberModel(2, 1, 5, 1));
+        /*
```

Comment:
Please add an explanation of why these are commented out e.g. "moved to CustomOptionsDialog" if you want to keep this code in